guides/v2.1/cloud/reference/git-integration.md

```diff
@@ -22,7 +22,7 @@ If you need help understand Git, you can review the following resources:
 *	[Git video](https://www.youtube.com/watch?v=8KCQe9Pm1kg){:target="\_blank"} with DevForge to understand how people use the repo and commands with a fun story
 
 ## Git CLI and clients {#clients}
-You can interact with Git using [CLI commands](https://git-scm.com/documentation){:target="\_blank"} or using a Git client. Git provides a <a href="https://git-scm.com/downloads" target="\_blank">Git client</a> option, or you can use other clients such as installed on your computer to be able to interact with {{site.data.var.ece}}.
+You can interact with Git using [CLI commands](https://git-scm.com/documentation){:target="\_blank"} or using a Git client. Git provides a [Git client](https://git-scm.com/downloads" target="\_blank) option, or you can use other clients such as installed on your computer to be able to interact with {{site.data.var.ece}}.
```

Comment:
This isn't the correct way to use a blank target in markdown - `{:target="_blank"}`